cmd/vic-machine/configure/configure.go

```diff
@@ -104,6 +104,13 @@ func (c *Configure) processParams() error {
 	return nil
 }
 
+// copyChangedConf should copy all configuration change specified by user to old configuration
+// currently we cannot automatically override old configuration with any difference in the new configuration, because there are some configuration set in VCH
+// creation process, for example, image store path, volume store path, network slot id, etc. So we'll copy changes based on user input
+func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n *config.VirtualContainerHostConfigSpec) {
+	//TODO: copy changed data
```

Comment:
Is there an open issue for this TODO? If so, please mention the issue number here for reference.